src/modules/multi-file-upload/services/multi.file.upload.service.js

```diff
@@ -142,11 +142,23 @@ const prepareStruct = ({ parentInfo, config, languageCode }, data) => {
         .then((response) => response.json())
         .catch(() => window.eZ.helpers.notification.showErrorNotification('Cannot get content type by identifier'))
         .then((response) => {
+            const fileValue = {
+                fileName: data.file.name,
+                data: data.fileReader.result.replace(/^.*;base64,/, ''),
+            }
```

Comment:
```suggestion
            };
```

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/292/commits/a9149aee84ef4eaa608ed4d6d2f288af6c58ef7f